content/projects/understanding-loops/_index.md

```diff
@@ -13,6 +13,23 @@ tags:
 title: Shopping cart calculations
 ---
 
+## Set up your environment
+
+### Javascript
+ 
+Your directory structure should look like this.
+ 
+```
+   >node_modules    <---- make sure this is in your .gitignore
+   >spec
+       > support
+           - jasmine.json
+       - shopping_cart_spec.js
+   >src
+       - shopping_cart.js
+   - package.json
+```
+
```

Comment:
Is this meant to be part of this PR? It's seems like its about a different thing....

Reply:
it's meant to be on another PR, mixed them up, oops. Its now fixed